be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandler.scala

```diff
@@ -67,8 +68,8 @@ class RollCallHandler(dbRef: => AskableActorRef) extends MessageHandler {
   def handleOpenRollCall(rpcRequest: JsonRpcRequest): GraphMessage = {
```

Comment:
Now that the function is a lot more readable, the logic seems a bit off to me.

Are you sure we're respecting the protocol ?

I may be missing something, but it seems that there's no check that the open message is actually targeting the right id. We're only checking that it's on the right channel. Same thing for the other roll call handlers.

Reply:
I believe that the checks are done within the validator, not the handler